docs/devices-api/more-info/webhooks.mdx

```diff
@@ -72,6 +76,63 @@ The format follows:
   </Expandable>
 </ResponseField>
 
+## Provider disconnected message
+
+Example payload:
+
+```json
+{
+  "meta": {
+    "messageId": "<message-id>",
+    "when": "<date-time-in-utc>",
+    "type": "devices.provider-disconnected"
+  },
+  "users": [
+    {
+      "userId": "951faef1-5cfd-464a-81f7-31f76edf309e",
+      "providers": ["fitbit", "cronometer"],
```

Comment:
This is an array only because of the admin endpoint, right? I'm thinking we should keep it consistent with the `provider-connected` WH. Maybe keep it a string and make the admin endpoint send multiple WH messages? Or change `provider-connected` to also be an array?

Just b/c from the dev's perspective, those could use the same "input schema/validation", just changing the WH type.

Reply:
This is also thinking ahead - since in theory a user can have multiple connected providers, in a single query we can try the refresh token on all of them, and multiple can fail.

As such, I'm thinking it makes sense to send a single WH to the cx with the user state after the query, rather than spamming them with multiple WHs.

Reply:
I understand, that's what I meant with "This is an array only because of the admin endpoint, right?"

If that's the case, from the cx's perspective, each time their user revoke access there's only one provider being disconnected. We'd be defining the format based on an admin endpoint that we could use _very_ often, if even again (unless I'm missing something).

But if you feel strong about it and we expect to use the admin endpoint often, should we change the `connected` one to match it? We kept the `connect` one with an array of users instead of a single entry also to keep the shape of the diff requests as similar as possible, when applicable.

Reply:
Discussed offline - will keep array and change connected one to have a one-item array